os/board/esp32_devkit/src/esp32_boot.c

```diff
@@ -115,6 +115,13 @@ void esp32_devKit_mount_partions(void)
 		}
 	}
 #endif
+
+#ifdef CONFIG_FS_ROMFS
+	ret = mount(CONFIG_ESP32_AUTOMOUNT_ROMFS_DEVNAME,CONFIG_ESP32_AUTOMOUNT_ROMFS_MOUNTPOINT,"romfs", 0, NULL);
```

Comment:
ret is defined under CONFIG_FLASH_PARITION , then it's rely on the dependency of these two CONFIG, be careful.

Reply:
good idea, fixed.